client/components/plans/plan/style.scss

```diff
@@ -169,3 +169,27 @@
 		left: -5px;
 		top: 2px;
 }
+
+.plan__free-plan-message {
+	@include breakpoint( "<660px" ) {
+		display: none;
+	}
+}
+
+.plan__free-domain-message {
+	color: $alert-green;
+	font-size: 13px;
+
+	@include breakpoint( "<660px" ) {
+		padding-left: 40px;
+	}
+
+	@include breakpoint( ">660px" ) {
```

Comment:
We need to include styles with breakpoints set at greater/less than `960px` based on `.has-sidebar`.